azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -528,6 +528,35 @@ func resourceApiManagementService() *schema.Resource {
 				Computed: true,
 			},
 
+			"tenant_access": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+						"id": {
```

Comment:
might be more clear if we add tenant here
```suggestion
						"tenant_id": {
```

Reply:
good idea